packages/directory/app/styles/navi-directory/components/dir-sidebar.less

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+ @text-blue: #3697f2;
```

Comment:
Maybe we should add this to the colors.less file in core. It seems like we're using it for more than just text

Reply:
Yes, I meant to change it. Will do.